code/modules/ruins/spaceruin_code/hilbertshotel.dm

```diff
@@ -228,24 +229,53 @@ GLOBAL_VAR_INIT(hhmysteryRoomNumber, 1337)
     do_sparks(3, FALSE, get_turf(arrived))
 
 /turf/closed/indestructible/hoteldoor
-    name = "Hotel Door"
-    icon_state = "hoteldoor"
-    explosion_block = INFINITY
-    var/obj/item/hilbertshotel/parentSphere
+	name = "Hotel Door"
```

Comment:
are these double indented or is it just GH messing with the showcase

Reply:
just Github